pytest_reportportal/listener.py

```diff
@@ -1,6 +1,9 @@
-import cgi
 import pytest
 import logging
+try:
```

Comment:
You can do import like:
```
try:
    from html import escape
except ImportError:
    from cgi import escape
```
And then remove module name like:
```
if report.longrepr:
    self.PyTestService.post_log(
        escape(report.longreprtext), loglevel='ERROR')
```
This removes not necessary code duplicity and exception handling.